tests/components/blueprint/test_websocket_api.py

```diff
@@ -80,3 +97,143 @@ async def test_import_blueprint(hass, aioclient_mock, hass_ws_client):
             },
         },
     }
+
+
+async def test_save_blueprint(hass, aioclient_mock, hass_ws_client):
+    """Test saving blueprints."""
+    raw_data = Path(
+        hass.config.path("blueprints/automation/test_event_service.yaml")
+    ).read_text()
+
+    with patch("pathlib.Path.write_text", return_value=Mock()) as write_mock:
```

Comment:
That return value is the default. 

```suggestion
    with patch("pathlib.Path.write_text") as write_mock:
```